addons/project/static/src/views/project_task_kanban/project_task_kanban_record.js

```diff
@@ -21,4 +23,46 @@ export class ProjectTaskRecord extends Record {
         }
         return context;
     }
+
+    modifyName(changes, new_values, element) {
+        new_values['name'] = `${new_values['name']} ${element}`;
+    }
+
+    modifyOtherFields(changes, new_values, element) {
+        this.modifyName(...arguments);
+    }
+
+    async _save() {
+        const changes = this.getChanges();
+        if (changes.name) {
+            let new_values = {name: ''};
+            let tags_names = new Set();
+
+            changes.name.split(' ').forEach(element => {
+                if (element[0] === '+' && element.length > 1)
+                    tags_names.add(element.substr(1));
+                else if (element === '@me') {
+                    if (!('user_ids' in changes))
+                        changes['user_ids'] = []
+
+                    changes['user_ids'].push(x2ManyCommands.linkTo(session.uid));
+                }
+                else if (element === '!')
+                    changes['priority'] = '1'
+                else
+                    this.modifyOtherFields(changes, new_values, element);
+            });
+
+            changes.name = new_values['name'].trim();
+            if (tags_names.size) {
+                const tags = Array.from(tags_names);
+                const existing_tags = await this.model.orm.call("project.tags", "search_read", [[['name', 'in', tags]]]);
+                const existing_tags_names = existing_tags.map(({name}) => name);
+                const tags_to_create = tags.filter(tag => !existing_tags_names.includes(tag));
+                changes['tag_ids'] = existing_tags.map(({id}) => x2ManyCommands.linkTo(id));
+                changes['tag_ids'] = changes['tag_ids'].concat(tags_to_create.map(tag => x2ManyCommands.create(false, { 'name': tag })));
+            }
+        }
+        return await super._save(...arguments, changes);
```

Comment:
Cannot we call update method to add the changes in the record instead of changing the signature of that method?